### PR TITLE
chore(host_metrics source): Adds cue docs for customizing sysfs and procfs in host_metrics component

### PR DESCRIPTION
**Pull request by eeyun**
Signed-off-by: Ian Henry <ianjhenry00@gmail.com>

### CODE REVIEW

File: docs/reference/components/sources/host_metrics.cue [hunk 1]
```diff
@@ -44,6 +44,24 @@ components: sources: host_metrics: {
 		platform_name: null
 	}
 
+    env_vars: {
+		PROCFS_ROOT: {
+			description: "Sets an arbitrary path to the system's Procfs root. Can be used to expose host metrics from within a container."
+			type: string: {
+				default: null
+				examples: "/mnt/host/proc"
```
Comment (binarylogic) [inline docs/reference/components/sources/host_metrics.cue:52]:
```suggestion
				examples: ["/mnt/host/proc"]
```

File: docs/reference/components/sources/host_metrics.cue [hunk 2]
```diff
@@ -44,6 +44,24 @@ components: sources: host_metrics: {
...
+			}
+		}
+
+		SYSFS_ROOT: {
+			description: "Sets an arbitrary path to the system's Sysfs root. Can be used to expose host metrics from within a container."
+			type: string: {
+                default: null
+				examples: "/mnt/host/sys"
```
Comment (binarylogic) [inline docs/reference/components/sources/host_metrics.cue:60]:
```suggestion
				examples: ["/mnt/host/sys"]
```

**Comment by binarylogic** [comment]
You'll need to run `cue fmt` to convert your spaces to tabs.

File: docs/reference/components/sources/host_metrics.cue [hunk 3]
```diff
@@ -44,6 +44,24 @@ components: sources: host_metrics: {
 		platform_name: null
 	}
 
+    env_vars: {
+		PROCFS_ROOT: {
+			description: "Sets an arbitrary path to the system's Procfs root. Can be used to expose host metrics from within a container."
+			type: string: {
+				default: null
```
Comment (bruceg) [inline docs/reference/components/sources/host_metrics.cue:51]: Should this default be `"/proc"` to describe the directory normally used?

File: docs/reference/components/sources/host_metrics.cue [hunk 4]
```diff
@@ -44,6 +44,24 @@ components: sources: host_metrics: {
...
+				examples: ["/mnt/host/proc"]
+			}
+		}
+
+		SYSFS_ROOT: {
+			description: "Sets an arbitrary path to the system's Sysfs root. Can be used to expose host metrics from within a container."
+			type: string: {
+                default: null
```
Comment (bruceg) [inline docs/reference/components/sources/host_metrics.cue:59]: Ditto `"/sys"`

**Review by bruceg — COMMENTED** [review]
Looks good, but I have a question about the defaults

File: docs/reference/components/sources/host_metrics.cue (hunk 3 above)
Comment (eeyun) [reply docs/reference/components/sources/host_metrics.cue:51]: So this was a consideration I also made but I think the answer is technically no? The documentation is for a envvar that's not normally set on a system so null is the default value. Maybe I ought to clarify that in the description?

**Comment by eeyun** [comment]
I've tried to correct the missing DCO here but doesn't seem to want to take. Anybody have any ideas?

**Comment by jamtur01** [comment]
@eeyun The `make signoff` command always seems to work for me.

**Comment by eeyun** [comment]
Well that certainly  seems to have fixed it  @jamtur01 :)

File: docs/reference/components/sources/host_metrics.cue (hunk 3 above)
Comment (bruceg) [reply docs/reference/components/sources/host_metrics.cue:52]: The same could be said for any optional value, but I don't think that's particularly useful. More useful would be to document the underlying value and what it defaults to if the setting is not present.

**Pull request merged**